core/app/[locale]/(default)/login/register-customer/_components/register-customer-form/index.tsx

```diff
@@ -188,17 +188,8 @@ export const RegisterCustomerForm = ({
 
     if (submit.status === 'success') {
       form.current?.reset();
-      setFormStatus({
-        status: 'success',
-        message: t('successMessage', {
-          firstName: submit.data?.firstName,
-          lastName: submit.data?.lastName,
-        }),
-      });
 
-      setTimeout(() => {
-        void login(formData);
-      }, 3000);
```

Comment:
This timeout hurt our register customer test, so I opted to remove it for now. I think we should be showing this message on the account page instead, but will do so in another PR.